### ISSUE TITLE
Layout buttons on title bar are not keyboard accessible 

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#143788]
Testing #143581

It seems I can't tab into the layout actions. If I click one of the button, then I can tab into that button but not others.

**Comment by rebornix** [comment]
![Recording 2022-02-23 at 10 30 54](https://user-images.githubusercontent.com/876920/155384901-cdc51287-64b2-467a-b500-0ac9ff89794c.gif)

This is what I was seeing. It's confusing that

* We can only tab into the first button
* The layout buttons are the very last one in the tab order, I expect it to be the first.

**Comment by sbatten** [comment]
I'm not sure if they should be first, cc @isidorn 

they are accessible via keyboard but this is not the most convenient way for a keyboard user to access these functions, so I feel they should be "out of the way"

**Comment by isidorn** [comment]
They should act like all our other toolbars. The whole toolbar is one tab-stop, pressing left / right navigates between the actions inside the toolbar.
And yes, they deserve one tab-stop, they should not be completely out of the way.

**Comment by sbatten** [comment]
The current behaviour is that which isi describes as correct
